src/plugins/pcr/src/import/ImportPrimersDialog.cpp

```diff
@@ -115,14 +115,17 @@ void ImportPrimersDialog::sl_selectionChanged() {
 }
 
 void ImportPrimersDialog::sl_contentChanged() {
-    buttonBox->button(QDialogButtonBox::Ok)->setEnabled(lwFiles->count() > 0);
+    buttonBox->button(QDialogButtonBox::Ok)->setEnabled(lwFiles->count() > 0 || lwObjects->count() > 0);
 }
 
 void ImportPrimersDialog::accept() {
     QList<Task*> tasks;
-    foreach (const QString& filePath, item2file) {
+    for (const QString& filePath : item2file) {
```

Comment:
Same here: use qAsConst with for & qt collections

Reply:
OK